cmd/builder/README.md

```diff
@@ -5,28 +5,28 @@ This program generates a custom OpenTelemetry Collector binary based on a given
 ## TL;DR
 
 ```console
-$ go install go.opentelemetry.io/collector/cmd/builder@v0.109.0
+$ go install go.opentelemetry.io/collector/cmd/builder@v0.130.0
```

Comment:
there is a known issue that is currently being patched w/ 0.130.0, i would suggest we recommend 0.129.0